### ISSUE TITLE
[Accessibility: Visual formatting] Unable to identify which template is selected when removing color from text and images

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#867]
To configure the browser to remove color from text:

For Firefox:
    Select Edit >Preferences
    On the Content tab, in the Fonts & Colors section, select the Colors… button.
    In the Colors dialog box, select the Use system colors checkbox, and set the Override the colors specified by the page with my selections above drop down to Always.
    Select OK to save changes.
    Select OK to exit the Options dialog box.

Then you will see there is no way to identify which template is selected when creating a new guest.

![307d7ad6f28ca964](https://cloud.githubusercontent.com/assets/4921718/13960002/36b16f8c-f035-11e5-9dda-0585c7ad520f.jpg)
